include/uapi/sound/sof/tokens.h

```diff
@@ -191,4 +192,9 @@
 /* COPIER */
 #define SOF_TKN_INTEL_COPIER_NODE_TYPE		1980
 
+/* pin_binding */
+#define SOF_TKN_PIN_BINDING_BIND_TYPE		2000
+#define SOF_TKN_PIN_BINDING_QUEUE_ID		2001
+#define SOF_TKN_PIN_BINDING_WNAME		2002
```

Comment:
I have no idea what "BINDING_WNAME" refers to.

Reply:
it is the widget name we are binding in a route, it may be source widget name or sink widget name.